### ISSUE TITLE
rename ktr reference to LGR

### ISSUE THREAD

**Issue by nickdesaulniers** [linux-3ds/linux#25]
open_agb_firm uses this reference, and I had noticed this as well from ifixit's teardown.
https://guide-images.cdn.ifixit.com/igi/OMYYisieWXRA2dBL.huge

**Comment by nickdesaulniers** [comment]
the bootloader needs to change, too.

**Comment by profi200** [comment]
Dunno if this should be renamed. KTR is the internal code name of the whole console/handheld. LGR only refers to the SoC itself. There are a bunch of other hardware changes on New 3DS than the SoC.

**Comment by nickdesaulniers** [comment]
[Screenshots from ifixit](https://guide-images.cdn.ifixit.com/igi/OMYYisieWXRA2dBL.huge) show that the SoC is marked LGR.  Unless the peripherals as described by the device tree (DT) need to be different between new 3DS, new 3DS XL, and new 2DS XL, then I suspect they are all using the same SoC.  The DT at this point only differs between amount of FCRAM and number of cores.  So I think LGR is more precise than KTR.

**Comment by profi200** [comment]
There are a few extra periphals not found on o3DS/CTR. See the red entries [here](https://www.3dbrew.org/wiki/IO_Registers). L2 cache and FCRAM are mentioned on the memory layout page. There are also a few changes outside the SoC like the improved parallax barrier, slightly better cameras, C-Stick and other small differences.

**Comment by nickdesaulniers** [comment]
Right, so the C-stick is a good example; does that exist on all 3 of new 3DS, new 3DS XL, and new 2DS XL? If so, then one DTS is fine; if not, then we'd consider breaking up the DTS into multiple, one for each unique board. Unless that's the case, the DTS should describe the SoC, not the board IMO.

**Comment by Wolfvak** [comment]
It exists for all the "new" systems and is controlled via a pretty simple I2C device (both CStick and ZL/ZR)
